app/services/decorators/capital_summary_decorator.rb

```diff
@@ -1,41 +1,60 @@
 module Decorators
   class CapitalSummaryDecorator
-    attr_reader :assessment
-
-    def initialize(record)
-      @record = record
-    end
-
-    def as_json # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-      return nil if @record.nil?
-
-      {
-        capital_items: {
-          liquid: liquid_items,
-          non_liquid: non_liquid_items,
-          vehicles: vehicles,
-          properties: {
-            main_home: PropertyDecorator.new(@record.main_home)&.as_json,
-            additional_properties: additional_properties
-          }
-        },
-        total_liquid: @record.total_liquid,
-        total_non_liquid: @record.total_non_liquid,
-        total_vehicle: @record.total_vehicle,
-        total_property: @record.total_property,
-        total_mortgage_allowance: @record.total_mortgage_allowance,
-        total_capital: @record.total_capital,
-        pensioner_capital_disregard: @record.pensioner_capital_disregard,
-        assessed_capital: @record.assessed_capital,
-        lower_threshold: @record.lower_threshold,
-        upper_threshold: @record.upper_threshold,
-        assessment_result: @record.assessment_result,
-        capital_contribution: @record.capital_contribution
-      }
+    def initialize(capital_summary)
+      @record = capital_summary
+    end
+
+    def as_json
+      payload_with_attrs unless @record.nil?
     end
 
     private
 
+    def payload
+      @payload ||= {}
+    end
+
+    def add_totals # rubocop:disable Metrics/AbcSize
+      payload[:total_liquid] = @record.total_liquid
```

Comment:
I'm not sure about this, my thoughts in a few of these decorators was to create some methods just to split the payload out a bit. But is it clearer? Not sure. I'm having to disable rubocop in either case so it might just be better clearer having as it was before but in a private method.